src/main/java/com/watch/shop/app/controller/WatchController.java

```diff
@@ -0,0 +1,57 @@
+package com.watch.shop.app.controller;
+
+import com.watch.shop.app.model.entity.Watch;
+import com.watch.shop.app.model.service.WatchService;
+import com.watch.shop.app.view.WatchView;
+import lombok.RequiredArgsConstructor;
+
+import static com.watch.shop.app.util.Constants.INVALID_CHOICE_MESSAGE;
+import static com.watch.shop.app.util.Constants.INVALID_INPUT_MESSAGE;
+
+@RequiredArgsConstructor
+public class WatchController {
+
+    private final WatchService watchService;
```

Comment:
this is ok and acceptable.

But as you see, you have 1 service in whole application, 1 view, and it's Watch application.
So we understand that view it's WatchView and service = WatchService

Classnames are ok, but variables could be:
private final WatchService service;
private final WatchView view;
private final MenuHandler menuHandler;

It would be a little bit easier to read.

IN a case we have a lot of service, of couse, then we should define:
WatchService watchService;
ShopService shopService;
SomeService someService;

But in our case, 1 service and 1 view, so it could be enough just view / service